### ISSUE TITLE
Discussion on code_of_conduct.md

### ISSUE THREAD

**Issue by burrrata** [pythonpete32/aragoncoop_governance#8]
It might make sense to have a few designated "community managers" who can help resolve disputes and address inappropriate conduct. That being said, checks and balances to power/influence need to be accounted for so that everyone feels represented and safe. How to do that without airing everyone's grievances in the #general channel of the group is an open question.

**Comment by burrrata** [comment]
Maybe worth reaching out to the A1 team to see how they do it? I think @light does a lot of that stuff, but honestly I'm not sure

**Comment by burrrata** [comment]
Reached out to @johnlight and @lkngtn on Keybase, so hopefully that helps. Not sure what other resources there are for CoC / community management stuff in the Aragon ecosystem though?

**Comment by pythonpete32** [comment]
I think a CM role is essential IMO. every community I have ever been a part of long-term has had a great manager

But this isn't a job for anyone and it will need a vote

**Comment by burrrata** [comment]
Yeah in most organizations there is a community manager or HR department who handles this. The Aragon Cooperative is a flat organization so this is not possible. We could either "wing it" (which hasn't worked out for us in the past), or someone could volunteer to do it and then be compensated for that via a weekly reward or something? Probably best to just ask the community what they think?

**Comment by burrrata** [comment]
The [Harassment Policy](https://github.com/hacklabto/hacklab-bylaws/blob/master/Anti-Harassment_Policy.mediawiki) for the hacklab might also be a good point of reference. It doesn't solve the lack of a Community Manager problem, but there's some good suggestions for how to handle disputes and confidentiality.

**Comment by pythonpete32** [comment]
I'll have a look
